### PR TITLE
Aut19 install dd agent

### PR DESCRIPTION
**Pull request by shinitiandrei**
Added datadog in Windows
- Installs datadog in Windows
- Unit and integration tests running and 100% covered.
- Minor refactor in recipes

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
 source 'https://supermarket.chef.io'
 
+cookbook 'datadog', '~> 2.6.0'
```
Comment (igorlg) [inline Berksfile]: Keep all dependencies on `metadata.rb`

File: test/unit/recipes/default_spec.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@
   end
 
   describe 'DataDog Agent as Docker Container' do
```
Comment (igorlg) [inline test/unit/recipes/default_spec.rb]: Split unit tests per recipe.

Comment (shinitiandrei) [inline test/unit/recipes/default_spec.rb]: Unit tests split!

File: Berksfile (hunk 1 above)
Comment (shinitiandrei) [inline Berksfile]: Removed as requested

**Pull request merged**